### ISSUE TITLE
Add to Man Page Or New Command Option

### ISSUE THREAD

**Issue by astroman68** [teejee2008/timeshift#750]
**Is your feature request related to a problem? Please describe.**
Sort-of. I use timeshift in BTRFS mode now for about the last 6 months, so this was a little new to me. I have timeshift setup to backup both volumes @ and @home and last night whilst screwing with file permissions I borked something in my home directory and couldn't log back in. So, I thought, no problem, just drop to tty and timeshift back an hour. Problem was, unknown to me, by default (as I much later discovered) by default timeshift only restores the root volume. Makes sense! Problem was, I looked under both the Man page and --help and found no information on how to get it to also restore @home. In the GUI there is an obvious check box when going through the restore wizard. So, with all that said, after I fixed my problem, I starting digging around to find out how to do it and discovered the /etc/timeshift/timeshift.json settings file with flag to change to make it restore home. It would be great if you guys could add a short blurb in the Man page to point the way.

**Describe the solution you'd like**
In Man Page add something to the effect:...

" To enable @home restore functionality at the command line, edit the timeshift.json file variable "include_btrfs_home_for_restore" to true located in /etc/timeshift/timeshift.json"

You guys could probably make something that sounds better.

**Describe alternatives you've considered**
Possibly add a command option that would allow you to add @home restore, that way you could leave the variable to false for safety reasons without having to go change it each time you want to restore @home.

**Additional context**
None. LOVE TIMESHIFT! SAVE ME MANY TIMES! THANKS FOR ALL YOU DO!

**Comment by tangeloservices** [comment]
i agree - in general i feel timeshift's GUI is fairly well documented and there are many, many articles on the internet using it via gui, but the terminal setup process and commands are lightly treated, and relatively undocumented. i am keen to help improve this.
